crimsobot/utils/tarot.py

```diff
@@ -168,6 +168,16 @@ async def reading(spread: str) -> Tuple[Optional[io.BytesIO], List[Tuple[str, st
         ]
         position_legend = ['PAST', 'PRESENT', 'FUTURE', 'REASON', 'POTENTIAL']
 
+    elif spread == 'one':
+        # a singular card on its own
+        # what is it? present?
+        bg_size = (w, h)
+        cards = await Deck.get_random_cards(1)
+        position = [
+            (0, 0)
+        ]
+        position_legend = ['PRESENT']  # just assuming once again
```

Comment:
https://discordapp.com/channels/552650672965943296/554799749593563137/727592213596340285
iudith said something about just keeping it empty, that's probably the best bet

Reply:
done, https://github.com/crimsobot/crimsoBOT/pull/73/commits/42655b5ebd17245dad2fd437645405dfa4e2340f